### PR TITLE
fix(rx-utils): createHandler union types fix

### LINKED ISSUE #183: new problem with create-handler
createHandler has problem with union types

### PR DESCRIPTION
**Pull request by scink**
fixes #183

### CODE REVIEW

File: packages/rx-utils/src/__tests__/create-handler.utils.spec.ts [hunk 1]
```diff
@@ -1,24 +1,42 @@
 import { createHandler } from '../create-handler.utils';
 import { Endomorphism, Lazy } from 'fp-ts/lib/function';
+import { constVoid } from 'fp-ts/es6/function';
```
Comment (raveclassic) [inline packages/rx-utils/src/__tests__/create-handler.utils.spec.ts]: please use `lib` version

File: packages/rx-utils/src/__tests__/create-handler.utils.spec.ts [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+
+type Effect<A> = (a: A) => void;
```
Comment (raveclassic) [inline packages/rx-utils/src/__tests__/create-handler.utils.spec.ts]: This is not an "Effect". If you need a type for void thunk you can use `interface VoidThunk<A> { (a: A): void; }`

Comment (scink) [reply packages/rx-utils/src/__tests__/create-handler.utils.spec.ts]: is it important for the `.spec` file?

Comment (scink) [reply packages/rx-utils/src/__tests__/create-handler.utils.spec.ts]: fixed

File: packages/rx-utils/src/__tests__/create-handler.utils.spec.ts (hunk 1 above)
Comment (scink) [reply packages/rx-utils/src/__tests__/create-handler.utils.spec.ts]: fixed

**Pull request merged**